index.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html> 
+<html lang="en">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+    <header class="first">
+        <div class="first-img"><img src="./images/Welcome.png" alt="welcome button at the top left"></div>
+        <div class="first-img" id="hamburger" > <img src="./images/Union.png" alt="a union of three vectors"> </div>
+    
+        <ul class="header-first">
+         <li>Portfolio</li>
+         <li>About </li> 
+         <li>Contact </li>
+        </ul>
+     
+
+    </header>
+    <br>
+    <section class="full-section hide" id="portfolio">
+        <div class="mobile-menu-section">
+            <div class="mobile-menu">
+                <div class="close-button" id="close" > <img src="./images/close.svg" alt="a union of three vectors"> </div>
+                <ul class="mobile-menu-ul">
+                    <li  class="mobile-menu-option"><a href="#portfolio">Portfolio</a></li>
+                    <li  class="mobile-menu-option"><a href="#about">About</a> </li> 
+                    <li  class="mobile-menu-option"><a href="#contact">Contact</a> </li>
+                   </ul>
+            </div>
+        </div>
+        <div class="dash"></div>
```

Comment:
- Kindly remove this part as it have not yet required to build desktop version or to implement Umberger button functionality  :+1: